packages/vscode-extension/src/webview/components/InspectDataMenu.tsx

```diff
@@ -1,10 +1,48 @@
+import React, { useState } from "react";
 import * as DropdownMenu from "@radix-ui/react-dropdown-menu";
 import { Frame, InspectDataStackItem } from "../../common/Project";
 import { DeviceProperties } from "../utilities/consts";
 import "./InspectDataMenu.css";
 
 type OnSelectedCallback = (item: InspectDataStackItem) => void;
 
+const MAX_INSPECT_ITEMS = 5;
+
+interface InspectItemProps {
+  item: InspectDataStackItem;
+  onSelected: (item: any) => void;
+  onHover: (item: any) => void;
+}
+
+const InspectItem = React.forwardRef<HTMLDivElement, InspectItemProps>(
+  ({ item, onSelected, onHover }, forwardedRef) => {
```

Comment:
Why `forwardRef`? As far as I can tell, it's not used

Reply:
I think it's the way for creating custom components in radix https://www.radix-ui.com/primitives/docs/components/dropdown-menu